src/flag_gems/utils/libentry.py

```diff
@@ -26,6 +26,29 @@
 version = triton.__version__.split(".")
 major_version, minor_version = eval(version[0]), eval(version[1])
 
+if major_version == 2:
+
+    def all_kwargs(self):
+        return {
+            **self.kwargs,
+            **{
+                k: getattr(self, k)
+                for k in (
+                    "num_warps",
+                    "num_ctas",
+                    "num_stages",
+                    "num_buffers_warp_spec",
+                    "num_consumer_groups",
+                    "reg_dec_producer",
+                    "reg_inc_consumer",
+                    "maxnreg",
+                )
+                if hasattr(self, k)
```

Comment:
I think it would be more clear and easier to verify these runtime params if we include a permlink to the triton codebase location where these params are actually used.